packages/devtools_app/lib/src/service/service_extensions.dart

```diff
@@ -327,10 +327,12 @@ final disableClipLayers = ToggleableServiceExtensionDescription<bool>._(
   disabledValue: false,
   gaScreenName: analytics_constants.performance,
   gaItem: analytics_constants.disableOpacityLayersOption,
-  description: 'Render all clipping effects during paint',
+  description: 'Render all clipping effects during paint.',
   tooltip: '''Disable this option to check whether excessive use of clipping is
 affecting performance. If performance improves with this option
 disabled, try to reduce the use of clipping effects in your app.''',
+  documentationUrl:
+      'https://docs.flutter.dev/development/tools/devtools/performance#more-debugging-options',
```

Comment:
these docs are added in @johnpryan's PR here: https://github.com/flutter/website/pull/6975